src/Datadog.Trace/Logging/LibLogScopeEventSubscriber.cs

```diff
@@ -39,7 +39,16 @@ public LibLogScopeEventSubscriber(IScopeManager scopeManager)
             }
             else
             {
-                SetDefaultValues();
```

Comment:
When the Tracer is first created AND `DD_LOGS_INJECTION=true`, this is invoked and causes the issues described.